### ISSUE TITLE
Fix errof on fedora-clang OS

### ISSUE THREAD

**Issue by ddotta** [ddotta/parquetize#52]
FYI @nbc 
There is an error on fedora-clang OS and it must be fixed before 2024-02-19 to safely retain your package on CRAN.

I'll try to take a look but I'm not sure I'll have the time in the meantime.

----

https://cran.r-project.org/web/checks/check_results_parquetize.html

<html><body>
<div class="container">
<h2>CRAN Package Check Results for Package <a href="https://cran.r-project.org/web/packages/parquetize/index.html"> <span class="CRAN"> parquetize </span> </a> </h2>
<p>
Last updated on 2024-01-24 09:57:55 CET.
</p>

Flavor | Version | Tinstall | Tcheck | Ttotal | Status | Flags
-- | -- | -- | -- | -- | -- | --
r-devel-linux-x86_64-debian-clang | 0.5.6.1 | 8.29 | 96.50 | 104.79 | NOTE |  
r-devel-linux-x86_64-debian-gcc | 0.5.6.1 | 6.99 | 73.08 | 80.07 | NOTE |  
r-devel-linux-x86_64-fedora-clang | 0.5.6.1 |   |   | 2.37 | ERROR |  
r-devel-linux-x86_64-fedora-gcc | 0.5.6.1 |   |   | 138.51 | NOTE |  
r-devel-windows-x86_64 | 0.5.6.1 | 9.00 | 107.00 | 116.00 | NOTE |  
r-patched-linux-x86_64 | 0.5.6.1 | 7.89 | 94.37 | 102.26 | OK |  
r-release-linux-x86_64 | 0.5.6.1 | 5.90 | 94.71 | 100.61 | OK |  
r-release-macos-arm64 | 0.5.6.1 |   |   | 60.00 | OK |  
r-release-macos-x86_64 | 0.5.6.1 |   |   | 114.00 | OK |  
r-release-windows-x86_64 | 0.5.6.1 | 11.00 | 120.00 | 131.00 | OK |  
r-oldrel-macos-arm64 | 0.5.6.1 |   |   | 56.00 | OK |  
r-oldrel-windows-x86_64 | 0.5.6.1 | 13.00 | 120.00 | 133.00 | OK |  

<h3>Check Details</h3>

<p>
Version: 0.5.6.1
<br>
Check: Rd \usage sections
<br>
Result: NOTE
<span style="font-family: monospace; white-space: pre;">
  Documented arguments not in \usage in Rd file 'expect_missing_argument.Rd':
    ‘message’
  
  Functions with \usage entries need to have the appropriate \alias
  entries, and all their arguments documented.
  The \usage entries must correspond to syntactically valid R code.
  See chapter ‘Writing R documentation files’ in the ‘Writing R
  Extensions’ manual.
</span>
Flavors: <a href="https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-clang/parquetize-00check.html"><span class="CRAN">r-devel-linux-x86_64-debian-clang</span></a>, <a href="https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-gcc/parquetize-00check.html"><span class="CRAN">r-devel-linux-x86_64-debian-gcc</span></a>, <a href="https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-gcc/parquetize-00check.html"><span class="CRAN">r-devel-linux-x86_64-fedora-gcc</span></a>, <a href="https://www.r-project.org/nosvn/R.check/r-devel-windows-x86_64/parquetize-00check.html"><span class="CRAN">r-devel-windows-x86_64</span></a>
</p>
<p>
Version: 0.5.6.1
<br>
Check: package dependencies
<br>
Result: ERROR
<span style="font-family: monospace; white-space: pre;">
  Package required but not available: ‘arrow’
  
  See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
  manual.
</span>
Flavor: <a href="https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-clang/parquetize-00check.html"><span class="CRAN">r-devel-linux-x86_64-fedora-clang</span></a>
</p>

</div>
</body>
</html>CRAN Package Check Results for Package parquetize

Last updated on 2024-01-24 09:57:55 CET.
Flavor 	Version 	Tinstall 	Tcheck 	Ttotal 	Status 	Flags
r-devel-linux-x86_64-debian-clang 	0.5.6.1 	8.29 	96.50 	104.79 	NOTE 	
r-devel-linux-x86_64-debian-gcc 	0.5.6.1 	6.99 	73.08 	80.07 	NOTE 	
r-devel-linux-x86_64-fedora-clang 	0.5.6.1 			2.37 	ERROR 	
r-devel-linux-x86_64-fedora-gcc 	0.5.6.1 			138.51 	NOTE 	
r-devel-windows-x86_64 	0.5.6.1 	9.00 	107.00 	116.00 	NOTE 	
r-patched-linux-x86_64 	0.5.6.1 	7.89 	94.37 	102.26 	OK 	
r-release-linux-x86_64 	0.5.6.1 	5.90 	94.71 	100.61 	OK 	
r-release-macos-arm64 	0.5.6.1 			60.00 	OK 	
r-release-macos-x86_64 	0.5.6.1 			114.00 	OK 	
r-release-windows-x86_64 	0.5.6.1 	11.00 	120.00 	131.00 	OK 	
r-oldrel-macos-arm64 	0.5.6.1 			56.00 	OK 	
r-oldrel-windows-x86_64 	0.5.6.1 	13.00 	120.00 	133.00 	OK 	
Check Details

Version: 0.5.6.1
Check: Rd \usage sections
Result: NOTE
  Documented arguments not in \usage in Rd file 'expect_missing_argument.Rd':
    ‘message’
  
  Functions with \usage entries need to have the appropriate \alias
  entries, and all their arguments documented.
  The \usage entries must correspond to syntactically valid R code.
  See chapter ‘Writing R documentation files’ in the ‘Writing R
  Extensions’ manual.
Flavors: r-devel-linux-x86_64-debian-clang, r-devel-linux-x86_64-debian-gcc, r-devel-linux-x86_64-fedora-gcc, r-devel-windows-x86_64

Version: 0.5.6.1
Check: package dependencies
Result: [ERROR](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-clang/parquetize-00check.html)
  Package required but not available: ‘arrow’
  
  See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
  manual.
Flavor: [r-devel-linux-x86_64-fedora-clang](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-clang/parquetize-00check.html)

**Comment by nbc** [comment]
Hi @ddotta ,

We can not fix this one, the problem is in the arrow package : https://cran.r-project.org/web/checks/check_results_arrow.html.

**Comment by ddotta** [comment]
Yes I saw this specific issue about this : https://github.com/apache/arrow/issues/39806
